internal/cmd/instance/deprovision_test.go

```diff
@@ -91,6 +93,24 @@ var _ = Describe("Deprovision command test", func() {
 		})
 	})
 
+	Context("when forceDelete flag is used", func() {
```

Comment:
maybe add a test that makes sure that if we don't use the flag the cascade and force params are not passed? (same for unbind)